OrbitCore/Capture.cpp

```diff
@@ -26,6 +26,7 @@
 #include "TestRemoteMessages.h"
 #include <fstream>
 #include <ostream>
+#include <cereal/types/vector.hpp>
```

Comment:
We should include "Serialization.h" in case we change the implementation at some point.